lib/compute/network.js

```diff
@@ -215,6 +221,137 @@ Network.prototype.createFirewall = function(name, config, callback) {
 };
 
 /**
+ * Create a subnetwork in this network.
+ *
+ * @resource [Subnetwork Resource]{@link https://cloud.google.com/compute/docs/reference/v1/subnetworks#resource}
+ * @resource [Subnetwork: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/subnetworks/insert}
+ *
+ * @param {string} name - Name of the subnetwork.
+ * @param {object} config - See a
+ *     [Subnetwork resource](https://cloud.google.com/compute/docs/reference/v1/subnetworks#resource).
+ * @param {string=} config.description - An optional description of this
```

Comment:
Remove this from the @&shy;params list-- we can let the `config` link to "Subnetwork resource" cover these other params. Generally, we only list in @&shy;params custom properties or ones that we simplify/override/change the behavior of.